src/hooks/useLazyComponent.tsx

```
@@ -0,0 +1,38 @@
+        <Spinner animation="border" variant="primary" className="mb-2" aria-hidden="true" />
+        <p className="mb-0">{loadingText}</p>
+      </div>
+    </div>
+  );
+  
+  // Return component with proper suspense wrapping
+  return (props: React.ComponentProps<T>) => (
+    <Suspense fallback={customFallback || defaultFallback}>
+      <LazyComponent {...props} />
+    </Suspense>
+  );
```

ESLint

[error] Component definition is missing display name

(react/display-name)